Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'rails', '~> 6.0.2.1'
 # Use postgresql as the database for Active Record
 gem 'pg', '>= 0.18', '< 2.0'
 # Use Puma as the app server
-gem 'puma', '~> 4.3.0'
+gem 'puma', '4.3.3'
```

Comment:
Don't we have some CI to do this also?
_(fix dependencies version and keep them up-to-date)_

Reply:
Yes we have renovate in this repo